ruby.rb

```diff
@@ -3,42 +3,66 @@
 
 # 1. Use TWO different Ruby methods to return a new array with all the numbers multiplied by 3. Expected output: [3, 6, 18, 27, 9, 63]
 
-myArray = [1, 2, 6, 9, 3, 21]
-
+# myArray = [1, 2, 6, 9, 3, 21]
 
+# # p myArray.map{|i| i*3}
+    
+#         # or 
+        
+# def multiply_by arr 
+#   p mult3 = arr.map {|el| el*3}
+# end 
+#   multiply_by myArray
 
```

Comment:
This is the same method used twice :) You could achieve the same effect with an each do loop.